doc/api/deprecations.md

```diff
@@ -142,17 +142,17 @@ undefined `digest` will throw a `TypeError`.
 <a id="DEP0010"></a>
 ### DEP0010: crypto.createCredentials
 
-Type: Runtime
+Type: End-of-Life
 
-The [`crypto.createCredentials()`][] API is deprecated. Please use
+The [`crypto.createCredentials()`][] API was removed. Please use
```

Comment:
It seems this link will not be valid after this PR, so we need to delinkify `` `crypto.createCredentials()` `` here.